db/migrate/20240929210828_create_megaphone_configs.rb

```diff
@@ -0,0 +1,13 @@
+class CreateMegaphoneConfigs < ActiveRecord::Migration[7.2]
+  def change
+    create_table :megaphone_configs do |t|
+      t.string :token
+      t.string :network_id
+      t.string :network_name
+      t.boolean "publish_enabled", default: false, null: false
+      t.bigint "feed_id", null: false
```

Comment:
I imagine we might want to consolidate these configs to a AR `store` field(s) at some point?  But... not much cost to having 2 tables for the 2 main integrations at the moment..

Reply:
yeah they were different enough I didn't want to tackle trying to make this polymorphic, but I agree, we could end up there.